src/event.rs

```diff
@@ -1363,14 +1376,28 @@ where
 				}
 			},
 			LdkEvent::ChannelReady {
-				channel_id, user_channel_id, counterparty_node_id, ..
+				channel_id,
+				user_channel_id,
+				counterparty_node_id,
+				funding_txo,
+				..
 			} => {
-				log_info!(
-					self.logger,
-					"Channel {} with counterparty {} ready to be used.",
-					channel_id,
-					counterparty_node_id,
-				);
+				if let Some(funding_txo) = funding_txo {
+					log_info!(
+						self.logger,
+						"Channel {} with counterparty {} ready to be used with funding_txo {}",
```

Comment:
While we're here, maybe we also want to log the user_channel_id?

Reply:
Doesn't seem we do that for other events, FWIW.

Reply:
Alright, fine by me